opentech/apply/funds/views.py

```diff
@@ -79,15 +79,17 @@ def form_valid(self, form):
                 'apply:submissions:determinations:form',
                 args=(form.instance.id,)) + "?action=" + action)
 
+        form.instance.perform_transition(action, self.request.user)
+
         response = super().form_valid(form)
         return self.progress_stage(form.instance) or response
 
     def progress_stage(self, instance):
-        proposal_transition = instance.get_transition('draft_proposal')
```

Comment:
Something to consider: I had to move progress_stage to the determination view: https://github.com/OpenTechFund/opentech.fund/pull/396/files#diff-69bf3b7f321569af3a40118621f7c5f3L82